android/src/main/java/com/swmansion/gesturehandler/core/LongPressGestureHandler.kt

```diff
@@ -53,43 +91,72 @@ class LongPressGestureHandler(context: Context) : GestureHandler<LongPressGestur
       previousTime = SystemClock.uptimeMillis()
       startTime = previousTime
       begin()
-      startX = sourceEvent.rawX
-      startY = sourceEvent.rawY
+      startX = getAverageX(sourceEvent)
+      startY = getAverageY(sourceEvent)
+
+      ++currentPointers
```

Comment:
From what I can see, in all but two random places we're using `++` as a suffix operator, can we stick to that?

Reply:
Sure, I'm just used to using pre-incrementation when possible since it is faster (however, I don't know if that's the case in Kotlin).

Done in [41495ce](https://github.com/software-mansion/react-native-gesture-handler/pull/3043/commits/41495cebe1a1286b315ea706470441d4b4c2141b)

Reply:
I'm 99% sure the compiler will figure it out 😄